Sprint-3/1-key-implement/1-get-angle-type.js

```diff
@@ -7,9 +7,19 @@
 // Write the code to pass the test
 // Then, write the next test! :) Go through this process until all the cases are implemented
 
+//we pass an angle between 0 to 360 to the function to see what angle name it is
+
 function getAngleType(angle) {
    if (angle === 90) return "Right angle";
    // read to the end, complete line 36, then pass your test here
+
+   else if (angle< 90) return "Acute angle";
+
+   else if(angle >90 && angle<180) return "Obtuse angle";
+
+   else if(angle === 180) return "Straight angle";
+
+   else if(angle > 180 && angle < 360) return "Reflex angle";
```

Comment:
you can also check additional  scenarios like what if number is greater than 360, I know it is not an ideal case for this function but  it would be good challenge for you to return a lets say "out of range" text if number is greater than 360